cpp_utils/include/cpp_utils/ReturnCode.hpp

```diff
@@ -30,36 +34,70 @@ namespace utils {
  *
  * It uses the fastdds ReturnCode_t
  */
-class ReturnCode : public eprosima::fastrtps::types::ReturnCode_t
+class ReturnCode
 {
 public:
 
-    //! Inherit Parent class constructors
-    using eprosima::fastrtps::types::ReturnCode_t::ReturnCode_t;
+    enum ReturnCodeValue
+    {
+        OK = 0,
```

Comment:
I do not remember if we decided to support only the return codes that we use in tools or all of them @juanlofer-eprosima

Reply:
I think we agreed to "reuse" only the generic ones, not the DDS specific. So perhaps we should also include "Unsupported", "OutOfResources", "Timeout" and "IllegalOperation". However we should thoroughly check in which cases we would be needing to convert a Fast-DDS ret code to a dev-utils one, maybe this parallelism is not required at all.

Reply:
I think the main idea was that our return code shouldn't be related to Fast's. Thus, I would only include Fast's non-DDS return codes if we find that we need them.